pyronear/models/ssresnet.py

```diff
@@ -0,0 +1,106 @@
+from torchvision.models.resnet import ResNet, resnet18, BasicBlock
+import torchvision
+from torch import nn
+import torch
+import torch.nn.functional as F
```

Comment:
Codacy seems to say that `resnet18`, `torchvision`, `import torch.nn.functional as F` an unused imports, you can remove it ;) (cf. the codacy check)